src/Client/pages/AdjectiveComparatives.fs

```diff
@@ -46,7 +53,7 @@ let view model dispatch =
 
         div [ Styles.middle ]
             [
-                div [ClassName "is-hidden-mobile"] 
+                FilterBlock.View.root model.FilterBlock (FilterBlock >> dispatch) 
```

Comment:
All filters are now wrapped in FilterBlock views, which is actually a pure component in case of desktop, but has a toggle button in mobile.